chef/data_bags/crowbar/bc-template-crowbar.json

```diff
@@ -29,19 +29,25 @@
   },
   "deployment": {
     "crowbar": {
-      "crowbar-revision": 0,
+      "crowbar-revision": 1,
       "crowbar-applied": false,
       "element_states": {
-        "crowbar": [ "all" ]
+        "crowbar": [ "all" ],
+        "crowbar-upgrade": [ "ready", "crowbar-upgrade" ]
```

Comment:
Not sure yet. Current wat it should work with only "crowbar-upgrade" here. 
That "ready" is for the case when we want to run the "crowbar-upgrade" role for the last time when reverting the node state back to ready. Not sure if this is enough or "applying" is needed, I've not tested it yet